benchexec/outputhandler.py

```diff
@@ -731,9 +734,11 @@ def add_result(self, run):
         self.counter += 1
         self.dic[run.category] += 1
         self.dic[(run.category, result.get_result_classification(run.status))] += 1
-        self.score += result.score_for_task(run.identifier, run.properties, run.category, run.status)
+        self.score += result.score_for_task(run.identifier, run.properties, run.category, run.status,
+                                            run.multiproperty_statuses)
         #if run.properties:
-        self.max_score += result.score_for_task(run.identifier, run.properties, result.CATEGORY_CORRECT, None)
+        self.max_score += result.score_for_task(run.identifier, run.properties, result.CATEGORY_CORRECT, None,
+                                                run.multiproperty_statuses)
```

Comment:
This is probably wrong. `max_score` does not depend on the actual tool output, which I presume is present in `run.multiproperty_statuses`.